rosco-web/config/rosco.yml

```diff
@@ -5,7 +5,9 @@ rosco:
   configDir: /some/path/to/config/packer
   jobs:
     local:
-      timeoutMinutes: 20
+      timeoutMinutes: 30
```

Comment:
@rguthriemsft Confirmed - Linux bake tested while having 'object_id' defined and undefined in spinnaker-local.yml and no issues found. Linux does not pass 'object_id' to Packer in either case.